apps/voting/contracts/Voting.sol

```diff
@@ -155,49 +159,64 @@ contract Voting is IForwarder, AragonApp {
     }
 
     function canVote(uint256 _voteId, address _voter) public view returns (bool) {
-        Vote storage vote = votes[_voteId];
+        Vote storage voteTmp = votes[_voteId];
 
-        return _isVoteOpen(vote) && token.balanceOfAt(_voter, vote.snapshotBlock) > 0;
+        return _isVoteOpen(voteTmp) && token.balanceOfAt(_voter, voteTmp.snapshotBlock) > 0;
     }
 
     function canExecute(uint256 _voteId) public view returns (bool) {
-        Vote storage vote = votes[_voteId];
+        Vote storage voteTmp = votes[_voteId];
```

Comment:
Is the rename of `vote` required so it doesn't clash with the `vote` function? The `voteTmp` name feels a bit off, specially when used below when we modify the storage by changing values in `voteTmp`.

Reply:
(and yes, I agree with your comment, but my rationale was that I was thinking of `voteTmp` as a pointer to an element in the array)

Reply:
Another typically used naming scheme is `vote_` for local variables.